pkg/common/constants.go

```diff
@@ -22,11 +22,17 @@ const (
 	HostName      = "host-name"
 	ActionContext = "actionContext"
 
-	SeataXidKey    = "SEATA_XID"
-	XidKey         = "TX_XID"
-	MdcXidKey      = "X-TX-XID"
-	MdcBranchIDKey = "X-TX-BRANCH-ID"
-	BranchTypeKey  = "TX_BRANCH_TYPE"
-	GlobalLockKey  = "TX_LOCK"
-	SeataFilterKey = "seataDubboFilter"
+	SeataXidKey     = "SEATA_XID"
+	XidKey          = "TX_XID"
+	XidKeyLowercase = "tx_xid"
+	MdcXidKey       = "X-TX-XID"
+	MdcBranchIDKey  = "X-TX-BRANCH-ID"
+	BranchTypeKey   = "TX_BRANCH_TYPE"
+	GlobalLockKey   = "TX_LOCK"
+	SeataFilterKey  = "seataDubboFilter"
+
+	// GrpcHeaderKey grpc header xid key
+	GrpcHeaderKey = XidKey
+	// GrpcHeaderKeyLowercase grpc header xid key lowercase
+	GrpcHeaderKeyLowercase = XidKeyLowercase
```

Comment:
> // GrpcHeaderKey grpc header xid key
GrpcHeaderKey = XidKey
// GrpcHeaderKeyLowercase grpc header xid key lowercase
GrpcHeaderKeyLowercase = XidKeyLowercase

can we delete these two ?